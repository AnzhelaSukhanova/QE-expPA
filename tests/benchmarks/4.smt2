(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 10))
(declare-fun z () (_ BitVec 10))
(declare-fun t () (_ BitVec 10))
(assert
 (exists ((x (_ BitVec 10))) (and 
 (or (bvult x (bvmul #b1111100101 y)) (= x (bvmul #b1111100101 y))) 
 (or (bvult z x) (= z x)) 
 (or (bvult x t) (= x t)))))
