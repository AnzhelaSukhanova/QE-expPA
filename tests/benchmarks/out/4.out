(and (not (bvult (bvmul y #b1111100101) z)) (not (bvult t z)))
